backend/controller/controller.go

```diff
@@ -647,14 +648,18 @@ func (s *Service) GetModuleContext(ctx context.Context, req *connect.Request[ftl
 	if err != nil {
 		return nil, connect.NewError(connect.CodeInternal, fmt.Errorf("could not get active schemas: %w", err))
 	}
-	schema, ok := slices.Find(schemas, func(s *schema.Module) bool { return s.Name == req.Msg.Module })
+	module, ok := slices.Find(schemas, func(s *schema.Module) bool { return s.Name == req.Msg.Module })
```

Comment:
Looks like configs, secrets and DSNs are now discovered without needing the declarations from a module. So the db query to find module can be skipped

Reply:
Now this module var is only used to get the module name, which can just get directly from the req

Reply:
Ah another thing on my mind: when should a deployment fail if a config/secret/DSN is missing. I think that was the other reason i was getting the module, so that we could fail earlier.

Reply:
Yeah it should fail at startup.